app/models/ontology/associations.rb

```diff
@@ -0,0 +1,23 @@
+module Ontology::Associations
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.